src/node.js

```diff
@@ -206,6 +207,13 @@ class Node {
             this.api.apiHost = addr.address
             this.api.apiPort = addr.port
 
+            if (gwmatch) {
+              this.gatewayAddr = gwmatch[2]
+              const addr = multiaddr(this.gatewayAddr).nodeAddress()
```

Comment:
Jumped some steps here :) Let me try again:

We currently have (with this PR):
```javascript
ipfs.apiHost
ipfs.apiPort
ipfs.gatewayHost
ipfs.gatewayPort
```

If we would expose the addresses as multiaddr, we should make sure the properties are consistent:
```javascript
ipfs.apiAddr
ipfs.gatewayAddr
```

So with that, I would get rid of `ipfs.apiHost` and `ipfs.apiPort`, and that would be the breaking change.

Does that make sense? We could also add both `ipfs.gatewayAddr` and `ipfs.apiAddr` in this PR, but then we'd have the `apiHost` and `apiPort` still there and I think that'd be slightly more confusion to have same thing as multiple properties.

Let me know what you think!